soc/espressif/esp32/memory.h

```diff
@@ -48,7 +48,7 @@
 	(SRAM1_SIZE - ((addr_dram) - SRAM1_DRAM_START) + SRAM1_IRAM_START)
 
 /* For safety margin between bootloader data section and startup stacks */
-#define BOOTLOADER_DRAM_SEG_LEN        0x6400
+#define BOOTLOADER_DRAM_SEG_LEN        0x7000
```

Comment:
Based on https://github.com/zephyrproject-rtos/zephyr/pull/73975#issuecomment-2183348010 having changed something and this only changing to give:
```
        iram_seg:       38960 B        40 KB     95.12%
        dram_seg:       26984 B      27392 B     98.51%
```
This does not enthrill confidence in my to allow this running in CI with such a tiny overhead for the build, any random change could happen with something in the zephyr kernel or anywhere for 408 bytes changing so would prefer these sizes be set larger to prevent this from silently breaking

Reply:
you are right, I missed that